Batch.py

```diff
@@ -1,29 +1,24 @@
 import torch
-from torchtext import data
```

Comment:
Not sure if it is a good idea to work with legacy modules.
Maybe working with the torch DataLoader could be a solution?

Reply:
Sure @na018 , feel free to edit the code!